### PR TITLE
RavenDB-13714 - Add RemoteFolderName to Glacier Backup

### PR DESCRIPTION
**Pull request by grisha-kotler**
https://issues.hibernatingrhinos.com/issue/RavenDB-13714

### CODE REVIEW

File: src/Raven.Server/Documents/PeriodicBackup/BackupTask.cs [hunk 1]
```diff
@@ -950,7 +950,7 @@ private static string Pluralize(int number)
         {
             using (var client = new RavenAwsGlacierClient(settings, progress, TaskCancelToken.Token))
             {
-                var key = CombinePathAndKey(_database.Name, folderName, fileName);
+                var key = CombinePathAndKey(settings.RemoteFolderName, folderName, fileName);
```
Comment (ppekrol) [inline src/Raven.Server/Documents/PeriodicBackup/BackupTask.cs]: I think this should be `settings.RemoteFolderName ?? _database.Name` to maintain backward compatibility

**Comment by ppekrol** [comment]
This also requires a bit of Studio work (it does not compile and we need to show that field in the UI)

File: src/Raven.Server/Documents/PeriodicBackup/BackupTask.cs (hunk 1 above)
Comment (grisha-kotler) [reply src/Raven.Server/Documents/PeriodicBackup/BackupTask.cs]: done

**Comment by grisha-kotler** [comment]
fixed and rebased

**Review by ppekrol — APPROVED** [review]

**Pull request merged**